README.md

```diff
@@ -45,7 +45,7 @@ This specific project holds the officialy supported Collaboration Tree backend.
 
 ## Technology
 
-TODO
+The technology is based on NGINX as proxy server, NaviServer as webserver, TCL-TK and Python3 as programing languages, PostgreSQL as database manager system and PSQL as SQL language script. 
```

Comment:
It's not worth blocking the PR, but you might want to separate the different technologies into a list and make them links to the official sites (ex. documentation) for each.  We get a lot of newer developers interested in this the cTree project, so it's helpful point contributors in the right direction.